ext/data/templates/default-anki-field-templates.handlebars

```diff
@@ -229,6 +229,25 @@
 {{/inline}}
 {{! End Pitch Accents }}
 
+{{#*inline "phonetic-transcriptions"}}
+    <ul>
```

Comment:
If empty, this generates `<ul></ul>`. Generating an empty value would be preferred, similar to the definition of `"frequencies"`.